<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeeDashboard</class>
 <widget class="QWidget" name="employeeDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Dashboard</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="navigationPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="inventoryButton">
        <property name="text">
         <string>Inventory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serviceReportButton">
        <property name="text">
         <string>Service Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="purchaseButton">
        <property name="text">
         <string>Purchase</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>473</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logOutButton">
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="inventoryFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="inventoryNavigation">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QPushButton" name="searchButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="searchBar">
           <property name="placeholderText">
            <string>Product Name...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="inventoryTitle">
           <property name="text">
            <string>INVENTORY</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="inventoryList">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="inventoryAttributes">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="productId">
              <property name="text">
               <string>Product ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="productName">
              <property name="text">
               <string>Product Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="quantity">
              <property name="text">
               <string>Quantity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="price">
              <property name="text">
               <string>Price</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="category">
              <property name="text">
               <string>Category</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QColumnView" name="products"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
